pkgs/top-level/stage.nix

```diff
@@ -243,6 +243,16 @@ let
       };
     } else throw "x86_64 Darwin package set can only be used on Darwin systems.";
 
+    # If already linux: the same package set unaltered
+    # Otherwise, return a natively built linux package set for the current cpu architecture string.
+    # (ABI and other details will be set to the default for the cpu/os pair)
+    pkgsLinux =
```

Comment:
I think this might be running into the same issue as when you were using `pkgsLocal` (https://github.com/NixOS/nixpkgs/pull/282401#discussion_r1461062860). ofborg might be happier if you exclude `pkgsLinux` from https://github.com/NixOS/nixpkgs/blob/1b439a273dadd4c675e81a0e05abbc9258889c02/pkgs/top-level/release-attrpaths-superset.nix